cunumeric/sort.py

```diff
@@ -97,12 +98,10 @@ def sort(output, input, argsort, axis=-1, stable=False):
     else:
         if axis is None:
             axis = 0
-        elif axis < 0:
-            axis = input.ndim + axis
-
-        if axis is not input.ndim - 1:
```

Comment:
This probably accidentally works due to Python's small-integer interning. But really I can't think of any reason this should not be a value comparison.